panic: failed to parse multiaddr "/ip4//tcp/9444"

https://github.com/incognitochain/incognito-chain/issues/634

If I pass "true" to the `--testnet`:

```
❯ ./incognito --enablewallet --wallet "wallet" --walletpassphrase "12345678" --testnet "true" --norpcauth --walletautoinit
NumCPU 4
2019-12-19 06:25:22.043 incognito.go:57 [INF] Server Log: Version 1.21.0-beta
2019-12-19 06:25:22.143 server.go:246 [INF] Server Log: Init 3-rd Party Random Client
encoded libp2p key: xx
2019-12-19 06:25:22.151 panic.go:679 [WRN] Server Log: Shutdown complete
panic: failed to parse multiaddr "/ip4//tcp/9444": invalid value "" for protocol ip4: failed to parse ip4 addr:

goroutine 1 [running]:
github.com/incognitochain/incognito-chain/peerv2.catchError(...)
	/Users/quanta/go/src/github.com/incognitochain/incognito-chain/peerv2/host.go:94
github.com/incognitochain/incognito-chain/peerv2.NewHost(0xc0004b3300, 0xb, 0xc0004b2360, 0x0, 0x24e4, 0x0, 0x0, 0x24e4)
	/Users/quanta/go/src/github.com/incognitochain/incognito-chain/peerv2/host.go:59 +0xa4b
main.(*Server).NewServer(0xc0000d3560, 0xc0004b2360, 0x5, 0x535bba0, 0xc000010160, 0x5347540, 0xc000230048, 0x5d00b60, 0xc0004b23b0, 0xb, ...)
	/Users/quanta/go/src/github.com/incognitochain/incognito-chain/server.go:278 +0xc69
main.mainMaster(0x0, 0x0, 0x0)
	/Users/quanta/go/src/github.com/incognitochain/incognito-chain/incognito.go:108 +0x651
main.main()
	/Users/quanta/go/src/github.com/incognitochain/incognito-chain/incognito.go:178 +0x241
```

--- a/peerv2/utils.go
+++ b/peerv2/utils.go
@@ -17,6 +17,9 @@ func ParseListenner(s, defaultIP string, defaultPort int) (string, int) {
 		if e != nil {
 			panic(e)
 		}
+		if splitStr[0] == "" {
+			return defaultIP, p
+		}
 		return splitStr[0], p
 	}
 	return splitStr[0], 0

